applications/nrf5340_audio/src/bluetooth/le_audio_cis_gateway.c

```diff
@@ -842,6 +843,72 @@ static struct bt_conn_cb conn_callbacks = {
 
 };
 
+/**
+ * @brief	Callback handler for play/pause.
+ *
+ * @note	This callback is called from MCS after receiving a
+ *		command from the client or the local media player.
+ *
+ * @param[in]	play  Boolean to indicate if the stream should be resumed or paused.
+ */
+static void le_audio_play_pause_cb(bool play)
+{
+	int ret;
+
+	LOG_DBG("Play/pause cb, state: %d", play);
+
+	if (play) {
+		playing_state = true;
+
+		if (audio_streams[AUDIO_CH_L].ep->status.state ==
+		    BT_AUDIO_EP_STATE_QOS_CONFIGURED) {
+			ret = bt_audio_stream_enable(&audio_streams[AUDIO_CH_L],
+						     lc3_preset_nrf5340.codec.meta,
+						     lc3_preset_nrf5340.codec.meta_count);
+
+			if (ret) {
+				LOG_WRN("Failed to enable left stream");
+			}
+		}
+
+#if !CONFIG_STREAM_BIDIRECTIONAL
+		if (audio_streams[AUDIO_CH_R].ep->status.state ==
+		    BT_AUDIO_EP_STATE_QOS_CONFIGURED) {
+			ret = bt_audio_stream_enable(&audio_streams[AUDIO_CH_R],
+						     lc3_preset_nrf5340.codec.meta,
+						     lc3_preset_nrf5340.codec.meta_count);
+
+			if (ret) {
+				LOG_WRN("Failed to enable right stream");
+			}
+		}
+#endif /* !CONFIG_STREAM_BIDIRECTIONAL */
+	} else {
+		playing_state = false;
```

Comment:
`playing_state` can be set directly to `play` outside the `if`

Reply:
Also, if something fails in these calls, should `playing_state` still be set to `true` ?

Reply:
You could have the situation where one stream is enabled and not the other, should it then stop everything? I decided to not ERR_CHK in here because there are a ton of edge cases where you hit play, but one headset suddenly disconnects  at the same time, for instance.